app/models/concerns/foreman_openscap/openscap_proxy_core_extensions.rb

```diff
@@ -33,10 +33,6 @@ def update_client_params(model_match, config)
       end
     end
 
-    def inherited_openscap_proxy_id
-      inherited_ancestry_attribute(:openscap_proxy_id)
-    end
```

Comment:
Why is this one removed? shouldn't it be similar to the hostgroup one?

Reply:
> Also removed method duplication from proxy_core_extensions, since the method is not relevant for hosts (the original inherited_ancestry_attribute is not defined on host) and already added to hostgroup by hostgroup_extensions.

@tbrisker, as I have mentioned earlier ^^^. I don't think the method was working for hosts and we weren't using it anywhere.

Reply:
oops, my bad.